python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ip_security.schema.yml

```
@@ -50,6 +50,14 @@ keys:
                 - str
               valid_values: [1, 2, 5, 14, 15, 16, 17, 20, 21, 24]
               description: Diffie-Hellman group for the key exchange.
+            integrity:
+              type: str
+              valid_values:
+                - sha1
```

vEOS running 4.32.1F and DCS-7280SRA running 4.33.0F both provide option to use `md5` as well:
```
avd-ci-leaf2(config-s-s1a-ipsec-ike)#integrity ?
  md5     Message Digest Algorithm 5
  sha1    Secure Hash Standard 1 (160 bit)
  sha256  Secure Hash Standard 2 (256 bit)
  sha384  Secure Hash Standard 2 (384 bit)
  sha512  Secure Hash Standard 2 (512 bit)
```

```
hss458...04:43:01(config-s-s6-ipsec-ike)#integrity ?
  md5     Message Digest Algorithm 5
  sha1    Secure Hash Standard 1 (160 bit)
  sha256  Secure Hash Standard 2 (256 bit)
  sha384  Secure Hash Standard 2 (384 bit)
  sha512  Secure Hash Standard 2 (512 bit)
```

```suggestion
                - md5
                - sha1
```